### PR TITLE
Fix CAN firmware problem on new code

### PR DESCRIPTION
**Pull request by mflanag**
Had to change enableMotorController function from writing to register 0x4f to 0xd0. It is supposed to write to 0xd0 for a can timeout setting but was writing to a motorPoleCount register which caused the motor to fuck up. Need to investigate how to properly set the can timeout value register. Wheels have been spun with this exact code :).

### CODE REVIEW

**Comment by RyanSchw** [comment]
`.gitignore` can be found in the [Template](https://github.com/PurdueElectricRacing/Template) repository. Please update this repository (or Template if we forgot stuff in Template) if you find that the `.gitignore` on this repo doesn't match the one on [Template](https://github.com/PurdueElectricRacing/Template).

File: Src/car.c [hunk 1]
```diff
@@ -242,16 +240,18 @@ void taskBlink(void* can)
 		}
 		if(!HAL_GPIO_ReadPin(P_AIR_STATUS_GPIO_Port,P_AIR_STATUS_Pin))
 		{
+			HAL_GPIO_TogglePin(LD5_GPIO_Port, LD5_Pin);
 			tx.Data[0] |= 0b00001000;
 		}
-		if (xSemaphoreTake(car.m_CAN, 100) == pdTRUE)
-		{
+		//if (xSemaphoreTake(car.m_CAN, 100) == pdTRUE)
```
Comment (RyanSchw) [inline Src/car.c]: Why is this commented?

**Review by RyanSchw — CHANGES_REQUESTED** [review]
Use (or update) the Template for `.gitignore` and fix the commented code

File: Src/car.c (hunk 1 above)
Comment (mflanag) [reply Src/car.c]: We don't use semaphores anymore. A queue is being used in it's place so there is no risk of it colliding on threads.

**Comment by mflanag** [comment]
The .gitignore is basically the same and it's good.

**Review by bsmrdel — APPROVED** [review]

File: Src/car.c (hunk 1 above)
Comment (RyanSchw) [reply Src/car.c]: We shouldn't be committing commented code. It's fine for now, we'll solve as part of #10. For future, this will not be allowed.

**Review by RyanSchw — APPROVED** [review]

**Comment by RyanSchw** [comment]
Thank you @mflanag for your descriptive PR, made it easier to review.

**Comment by mflanag** [comment]
no problem, i will try to conform more next time, sorry

**Pull request merged**